cluster/cluster.go

```diff
@@ -161,16 +172,35 @@ func (c *Cluster) DeployControlPlane(ctx context.Context, svcOptionData map[stri
 			controlPlaneHosts = append(controlPlaneHosts, host)
 		}
 	}
-	if err := services.UpgradeControlPlaneNodes(ctx, kubeClient, controlPlaneHosts,
+
+	for _, host := range notReadyHosts {
+		notReadyHostNames = append(notReadyHostNames, host.HostnameOverride)
+	}
+	// attempt upgrade on NotReady hosts without respecting maxUnavailable
+	logrus.Infof("Attempting to redeploy controlplane components on following hosts in NotReady status: %v", strings.Join(notReadyHostNames, ","))
```

Comment:
it's weird this comment says something different than what we log, redeploy is also different than upgrade.